message MessageRequestGetToken
{
    required int64 index        = 1;
    required string request_id  = 2;
    required int64 block_id     = 3;
    required int64 client_id    = 4;
}
